### ISSUE TITLE
Timeline state information is conveyed through graphical icons only

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4602]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 1.3.3 Sensory Characteristics

**Test Environment:**
- Test URL: https://clarity.design/documentation/timelines
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0
- Screen reader: NVDA 19.3.1

**Steps to reproduce:**
1. Open the test URL.
2. Select any one of the tabs provided for horizontal, vertical and angular timelines.
3. Observe the states provided for each step in timeline.

**Expected result:**
State of each step should be conveyed using visible text for representation of graphical symbol.

**Actual result:**
Only graphical symbol is used to convey state of each step in the timeline.

**Reference:**
https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/F26

**Proposed solution:**
Place the textual information closely to the graphic symbols so user can relate it to the symbolic representation. It can be placed above or below or on either sides adjacently. Cannot utilize tool-tip here as the symbols are not clickable so keyboard user cannot focus it see the tool-tip.

**Reporter Email:** gkhadke@vmware.com

**Comment by gnomeontherun** [comment]
Hi there, this is an automated message.

In an effort to dedupe and track accessibility reports consistently, this issue is being closed and tracked internally with our Accessibility team. It will still be triaged and addressed as part of ongoing work.
